GUI/CalibrateTaskWindow.xaml.cs

```diff
@@ -228,12 +228,13 @@ private void SaveButton_Click(object sender, RoutedEventArgs e)
 
             CommonParameters CommonParamsToSave = (TheTask.CommonParameters as CommonParameters).Clone();
 
-            DigestionParams digestionParamsToSave = new DigestionParams();
-            digestionParamsToSave.MaxMissedCleavages = int.Parse(missedCleavagesTextBox.Text, CultureInfo.InvariantCulture);
-            digestionParamsToSave.MinPeptideLength = int.Parse(txtMinPeptideLength.Text, NumberStyles.Any, CultureInfo.InvariantCulture);
-            digestionParamsToSave.MaxPeptideLength = int.Parse(txtMaxPeptideLength.Text, NumberStyles.Any, CultureInfo.InvariantCulture);
-            digestionParamsToSave.Protease = (Protease)proteaseComboBox.SelectedItem;
-            digestionParamsToSave.MaxModificationIsoforms = int.Parse(maxModificationIsoformsTextBox.Text, CultureInfo.InvariantCulture);
+           
+            Protease protease = (Protease)proteaseComboBox.SelectedItem;
+            int MaxMissedCleavages = int.Parse(missedCleavagesTextBox.Text, CultureInfo.InvariantCulture);
+            int MinPeptideLength = int.Parse(txtMinPeptideLength.Text, NumberStyles.Any, CultureInfo.InvariantCulture);
+            int MaxPeptideLength= int.Parse(txtMaxPeptideLength.Text, NumberStyles.Any, CultureInfo.InvariantCulture);
+            int MaxModificationIsoforms= int.Parse(maxModificationIsoformsTextBox.Text, CultureInfo.InvariantCulture);
+            DigestionParams digestionParamsToSave = new DigestionParams(protease.Name, MaxMissedCleavages, MinPeptideLength, MaxPeptideLength, MaxModificationIsoforms);
```

Comment:
make sure you're including the specifiers in all of these:
`new DigestionParams(Protease: protease.Name, ........`